src/style.css

```diff
@@ -0,0 +1,90 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  margin: 0 auto;
+}
+
+.searchForm {
+  margin-top: 20px;
+  position: relative;
+  border: none;
+}
+
+.searchInput {
+  height: 70px;
+  width: 100%;
+  padding-left: 62px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  font-family: inherit;
+  color: #3d4e61;
+  font-weight: 800;
+  font-size: 16px;
+}
+
+.searchInputSmall {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+}
+
+.searchInput::placeholder {
+  font-family: inherit;
+  color: #3d4e61;
```

Comment:
create variables for repeated colors

Reply:
the same 
https://github.com/mate-academy/layout_search-bar-airbnb/pull/3121/files#r1145213280